src/Components/WebAssembly/WebAssembly/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -138,90 +160,6 @@ protected override void HandleException(Exception exception)
             }
         }
 
-        /// <inheritdoc />
-        public override Task DispatchEventAsync(ulong eventHandlerId, EventFieldInfo? eventFieldInfo, EventArgs eventArgs)
```

Comment:
Can you help me understand what the consequences of removing this API are?

Reply:
The API remains, it's just this override that is no longer needed, since the base class implementation is now sufficient.